### PR TITLE
ACL per interface

### PR DESCRIPTION
**Pull request by gthess**

### CODE REVIEW

File: daemon/acl_list.c [hunk 1]
```diff
@@ -466,7 +549,223 @@ acl_list_apply_cfg(struct acl_list* acl, struct config_file* cfg,
...
+}
+
+void
+acl_interface_init(struct acl_list* acl_interface)
+{
+	regional_free_all(acl_interface->region);
+	/* We want comparison in the tree to include only address and port.
+	 * We don't care about comparing node->net. All addresses in the
+	 * acl_interface->tree should have either 32 (ipv4) or 128 (ipv6).
+	 * Initialise with the appropriate compare fucntion but keep treating
```
Comment (wcawijngaards) [inline daemon/acl_list.c]:
```suggestion
	 * Initialise with the appropriate compare function but keep treating
```

File: daemon/acl_list.h [hunk 2]
```diff
@@ -119,6 +119,20 @@ struct acl_list* acl_list_create(void);
  */
 void acl_list_delete(struct acl_list* acl);
 
+/**
+ * Insert interface in the alc_list. This should happen when the listening
```
Comment (wcawijngaards) [inline daemon/acl_list.h]:
```suggestion
 * Insert interface in the acl_list. This should happen when the listening
```

**Review by wcawijngaards — APPROVED** [review]
The changes look good to me. Small spelling fixes. Nice to have an extensive test.

**Comment by gthess** [comment]
Thanks for the review @wcawijngaards!

**Pull request merged**